WeakAurasOptions/OptionsFrames/TextEditor.lua

```diff
@@ -92,6 +94,103 @@ local function set_scheme()
   color_scheme["not"] = theme["Logical"]
 end
 
+-- Define the premade snippets
+local premadeSnippets = {
```

Comment:
I think, these snippets would benefit from a tagging system in which editor they should be. Though maybe that's more for version 2 than for version 1.

Reply:
That would be cool. Also probably search could go in a version2. Guess we can see if it takes off and people start making lots of their own snippets to need it first. 
My first version of that table was a straight [name] = "string" table, but giving it the ability to have more info in each snippet seemed worth reorganising the table for now.

Reply:
It would also be a good idea for the built-in snippets, as currently every snippet is shown everywhere.